crux-core/src/crux/query.clj

```diff
@@ -1388,7 +1387,7 @@
                                                                        var->joins
                                                                        type->clauses
                                                                        in-vars
-                                                                       stats)
+                                                                       db)
```

Comment:
Bit unsure here, so likely fine, but the entire idea of the encode-value-fn is to clearly separate compile time from runtime of the query, and not use the index-snapshot during the former.

Reply:
Ah, ok, thanks - seems a quick chat will be best to resolve this one?

Reply:
done :smile: